js-pkg/packages/driftdb-react/src/index.tsx

```diff
@@ -255,6 +260,10 @@ export function useSharedReducer<State, Action>(
     })
   }
 
+  useEffect(() => {
+    reducerRef.current!.subscribe()
+  }, [reducerRef.current])
+
```

Comment:
This also needs a teardown function. (As does the presenceListener's useEffect further down)